content/applications/sales/sales/send_quotations/quote_template.rst

```diff
@@ -1,93 +1,120 @@
-products and quantities as well as the expiration time of the quotation.
+To create a new quotation template, click the :guilabel:`Create` button, located in the upper-left
+corner. Doing so reveals a blank quotation template form that can be customized in a number of ways.
 
-.. image:: quote_template/quotations_templates_3.png
+.. image:: quote_template/blank-quotation-template.png
    :align: center
-   :class: img-thumbnail
-   :alt: Create a new quotation template on Odoo Sales
+   :alt: Create a new quotation template on Odoo Sales.
 
-Design your template
-====================
+Start by entering a name for the template in the :guilabel:`Quotation Template` field. Then, in the
+:guilabel:`Quotation expires after` field, designate how many days the quotation template will
+remain valid for, or leave the field on the default `0` to keep the template valid indefinitely.
 
-You will have the possibility to design your template and edit the customer interface in order
-to manage what clients will see before accepting and paying the quotation. For example, you will
-be able to describe your company, your services and your products. To do so, you can click on
-**Edit** and you will be brought to the quotation builder.
+In the :guilabel:`Company` field, designate to which company this quotation template applies, if
+working in a multi-company environment.
 
-.. image:: quote_template/quotations_templates_4.png
-   :align: center
-   :class: img-thumbnail
-   :alt: Design your quotation template on Odoo Sales
+Beneath those fields are three tabs: :guilabel:`Lines`, :guilabel:`Optional Products`,
+:guilabel:`Confirmation`.
+
+In the :guilabel:`Lines` tab, products can be added to the quotation template.
 
-You can easily edit the content of your template by dragging & dropping different building blocks
-to organize your quotation. For example, you can add a content block to describe your products.
+In the :guilabel:`Optional Products` tab, optional products can be added to the quotation template.
 
-.. image:: quote_template/quotations_templates_5.png
+Lastly, in the :guilabel:`Confirmation` tab, the options to require an :guilabel:`Online Signature`
+and/or an :guilabel:`Online Payment` to confirm orders automatically are available.
+
+There's also an option to add a :guilabel:`Confirmation Mail` to the quotation template, as well -
+meaning whenever this template is used, and an order is confirmed, an email is sent to the customer
+informing them that their order has been confirmed.
+
+.. image:: quote_template/quotations-templates-confirmation-tab.png
    :align: center
-   :class: img-thumbnail
-   :alt: Drag & drop building blocks to create your quotation template on Odoo Sales
+   :alt: Allow customers to sign electronically or to pay online on Odoo Sales.
 
-Use a quotation template
-========================
+Design quotation templates
+==========================
```

Comment:
need a **Note** here since this section is not applicable without the feature enabled (will throw a DB error and/or button will be missing)

I created an internal anchor for easy navigation here, like the [example on the rst cheat sheet](https://www.odoo.com/documentation/15.0/contributing/documentation/rst_cheat_sheet.html#custom-anchors).

```suggestion
==========================

.. note::
   The following section on designing quotation templates only applies to databases that have the
   *Quotation Builder* feature enabled, which is accessible by following the :ref:`configuration
   steps above <sales/send_quotations/templates>`.
```